fusequery/query/src/optimizers/mod.rs

```diff
@@ -4,12 +4,15 @@
 
 mod optimizer_filter_push_down_test;
 mod optimizer_limit_push_down_test;
+mod optimizer_projection_push_down_test;
```

Comment:
Let's begin to use `#[cfg(test)]` for test mods in this pr.

Reply:
About `#[cfg(test)]` test organization is here:
https://doc.rust-lang.org/book/ch11-03-test-organization.html